_data/accessibility-tests/icon-list.yml

```diff
@@ -0,0 +1,49 @@
+title: Icon list
+component_status: pass
+test_items:
+# General tests
+  - summary: Descriptive alt text identifies icon purpose.
+    summary_additional: When you view icons in an icon list component, you can mouse over an icon to see its meaning or purpose. Decorative icons will have no alt text.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.1.1
+  - summary: Color alone is not used to convey information.
+    summary_additional: When you view an icon list, you never have to rely on color alone to understand the meaning of the icons being used.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.1
+  - summary: Icons meet color contrast requirements.
+    summary_additional: When you look at an icon list and use ANDI or another color contrast analyzer to look at the hex numbers, contrast between the icons and the background is at least 3:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.11
+  - summary: Icon list content is written in plain language.
+    summary_additional: When you read the content in the icon list, the information is easy to understand. Content is written at the appropriate level for the audience.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 3.1.5
+  - summary: Icons used in an icon list are used the same way across the site.
+    summary_additional: When you see or tab to any icon list, icons used will always be represent the same thing in other lists on other parts of the website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 3.2.4
+# Zoom/screen magnification tests
+  - summary: Text and icons retain functionality when screen is magnified.
+    summary_additional: When you zoom in to 200%, icons and text are readable and retain all of their functionality.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.4
+# Screen reader tests
+  - summary: Screen reader announces icon meaning and purpose.
```

Comment:
**suggestion:** Recommend we added "when needed" here to signal that this check isn't always needed. @finekatie @alex-hull, any objections?
```suggestion
  - summary: Screen reader announces icon meaning and purpose when needed.
```

Reply:
I wonder if maybe the "when needed" part is in the gherkin body and not the header? I feel like it's a little vague. I may also have been looking at it too much?

Reply:
@finekatie So this component _is_ icons so they are always present. It's only a matter of if they're decorative or if they provide meaning.

Also I like the idea of it being in the summary so it's top level. That way people will know early if it's relevant or not

Reply:
Resolved in [17213b3](https://github.com/uswds/uswds-site/pull/3008/commits/17213b3185c08cbf22705e6e350d52dba6372131).

I kept Amy's initial recommendation of adding "when needed" to the summary rather than updating the `summary_additional` since Charlie mentioned this is the icon component (so they'd always be present). I think that makes sense too!

Reply:
looks good to me!